tests/reconfig/scripts/cafe-routes.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: cafe-tls-redirect
 spec:
   parentRefs:
-  - name: gateway.networking.k8s.io/v1
+  - name: gateway
```

Comment:
huh, I swear I fixed this when I ran the tests last time.

Reply:
Gotcha. Once all the tests are automated, we'll likely be "resetting" and only comparing results going forward from there.